### ISSUE TITLE
Delete confirmation

### ISSUE THREAD

**Issue by Shuunen** [pierrerousseau/zero-feeds#10]
Hi,

Thanks for the app :) really handful !

But the delete button is so close to the feed title that I delete many feeds by mistake, maybe moving the button elsewhere or add a confirm will avoid this.

**Comment by pierrerousseau** [comment]
Hi, thanks for using the app!
I'm planning to remove the left menu and so this problem will be solved ... but i have to admit that it will not be very soon ... 
When you "remove" a feed it should then go to the add form and so you can add it again without too much effort, does it work and can you live with that for a few months (2 or 3) ? If not, i'll try to add the confirm box as a first fix.

**Comment by Shuunen** [comment]
Yes I totally can live without :) and yes the redirect to add form can help restoring. So we will see in the next version :)
